packages/server-wallet/server-wallet.api.md

```diff
@@ -167,10 +161,6 @@ export interface EngineInterface {
     // (undocumented)
     pushUpdate(m: unknown): Promise<SingleChannelOutput>;
     // (undocumented)
-    registerAppBytecode(appDefinition: string, bytecode: string): Promise<void>;
```

Comment:
This removal needs to be addressed.

Reply:
Right now the wallet exposes a `registerAppDefinition` which can be used to register bytecode for an app.

Reply:
ohh -- this is the API change for the _engine_. I thought that it was for the _wallet_, and therefore believed that `registerAppDefinition` was removed from the wallet API.